openedx/core/apidocs.py

```diff
@@ -12,3 +13,44 @@
     email="oscm@edx.org",
     #license=openapi.License(name="BSD License"),                       # TODO: What does this mean?
 )
+
+
+def cursor_paginate_serializer(inner_serializer_class):
```

Comment:
FYI @nedbat 

This is my current workaround for the issue you identified several months ago in which Swagger won't always show the paginated version of a schema.